### PR TITLE
add new target StellarF7 Dual HD

### PR DESCRIPTION
**Pull request by cvetaevvitaliy**
Add new target StellarF7 Dual HD

### CODE REVIEW

File: configs/STELLARF7_DUAL/config.h [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+#define TIMER_PIN_MAPPING TIMER_PIN_MAP( 0, PC9, 2, 0 ) \
+                          TIMER_PIN_MAP( 1, PC8, 2, 1 ) \
+                          TIMER_PIN_MAP( 2, PC7, 2, 1 ) \
+                          TIMER_PIN_MAP( 3, PC6, 2, 1 ) \
+                          TIMER_PIN_MAP( 4, PB7, 1, 0 ) \
+                          TIMER_PIN_MAP( 5, PB6, 1, 0 ) \
+                          TIMER_PIN_MAP( 6, PB5, 1, 0 ) \
+                          TIMER_PIN_MAP( 7, PB4, 1, 0 ) \
+                          TIMER_PIN_MAP( 8, PB3, 1, 0 ) \
```
Comment (haslinghuis) [inline configs/STELLARF7_DUAL/config.h:89]: Servo does not need DMA.
```suggestion
                          TIMER_PIN_MAP( 8, PB3, 1, -1) \
```

File: configs/STELLARF7_DUAL/config.h [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+#define DEFAULT_DSHOT_BURST DSHOT_DMAR_ON
+//TODO #define MOTOR_PWM_PROTOCOL DSHOT300
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SCALE 110
+#define DEFAULT_CURRENT_METER_SCALE 182
+#define BEEPER_INVERTED
+//set osd_displayport_device = MAX7456
+#define MAX7456_SPI_INSTANCE SPI2
+#define FLASH_SPI_INSTANCE SPI2
```
Comment (haslinghuis) [inline configs/STELLARF7_DUAL/config.h:112]: Sharing SPI

File: configs/STELLARF7_DUAL/config.h [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+#define ADC1_DMA_OPT 1
+
+#define SERIALRX_UART SERIAL_PORT_USART2
+
+//TODO beacon RX_LOST
+//TODO beacon RX_SET
+
+#define MAG_I2C_INSTANCE (I2CDEV_1)
+#define BARO_I2C_INSTANCE (I2CDEV_1)
+#define ADC_INSTANCE ADC1
```
Comment (haslinghuis) [inline configs/STELLARF7_DUAL/config.h:101]: ADC1 is default instance.
```suggestion
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/STELLARF7_DUAL/config.h (hunk 2 above)
Comment (sugaarK) [reply configs/STELLARF7_DUAL/config.h:112]: > Sharing SPI

@cvetaevvitaliy  we don't allow this

File: configs/STELLARF7_DUAL_HD/config.h [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+#define SPI2_SCK_PIN PB13
+#define SPI1_SDI_PIN PA6
+#define SPI2_SDI_PIN PB14
+#define SPI1_SDO_PIN PA7
+#define SPI2_SDO_PIN PB15
+#define ADC_VBAT_PIN PC0
+#define ADC_CURR_PIN PC1
+#define PINIO1_PIN PA15
+#define FLASH_CS_PIN PC13
+#define MAX7456_SPI_CS_PIN PC3
```
Comment (haslinghuis) [inline configs/STELLARF7_DUAL_HD/config.h:70]: MAX7456 has been removed
```suggestion
```

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**